packages/gemini/gemini.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Carmelo Piccione <carmelo.piccione@gmail.com>"
+synopsis: "OCaml bindings for Gemini Trading Exchange API"
+description: "This library implements the Gemini exchange v1 REST, Market
+Data web, and Order events websockets services. It is backed by cohttp-async
+and websockets-async to do the heavy lifting. A provisional console interface
+is also provided using s-expressions to encode request parameters."
+authors: "Carmelo Piccione"
+homepage: "https://github.com/struktured/ocaml-gemini"
+license: "MIT"
+bug-reports: "https://github.com/struktured/ocaml-gemini/issues"
+dev-repo: "git+https://github.com/struktured/ocaml-gemini.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+doc: "https://struktured.github.io/ocaml-gemini/"
+depends: [
+  "ocaml" {>= "4.06.1"}
```

Comment:
Is this overly strict or the package is actually using new features from ocaml 4.06? I think in that case already 4.06.0 should be enough.

Reply:
It probably is overly strict but I never tested with it 4.06.0. Do you want me try it and see if it works?